shared/login/login/index.js

```diff
@@ -75,7 +80,8 @@ export default connect(
       onForgotPassphrase: () => dispatch(openAccountResetPage()),
       onLogin: (user, passphrase, store) => dispatch(relogin(user, passphrase, store)),
       onSignup: () => dispatch(routeAppend(['signup'])),
-      onSomeoneElse: () => { dispatch(login()) }
+      onSomeoneElse: () => { dispatch(login()) },
+      onSaveInKeychainChange: (user, saveInKeychain) => { if (!saveInKeychain) dispatch(clearStoredSecret(user)) }
```

Comment:
actually i think its cleaner to make saveInKeychain a param to the action, then have it do nothing in the action (we could do something else later. either way the component shouldn't know about what needs to happen)
so something like
```
dispatch(storedSecretChanged(user, saveInKeychain)
```